resources/services/Auth.php

```diff
@@ -65,7 +65,14 @@ private static function remember_me()
     private static function decrypt_cookie()
     {
         if (self::$request->cookies->get('zype_wp')) {
-            self::$cookie = JWT::decode(self::$request->validateCookie('zype_wp', ['textfield']), Config::get('zype.cookie_key'), array('HS256'));
+            try {
+                self::$cookie = JWT::decode("adsd", Config::get('zype.cookie_key'), array('HS256'));
+            } catch (\UnexpectedValueException $e) {
+                if($e->getMessage() == 'Wrong number of segments') {
```

Comment:
Why are we limiting this against this particular error message only? Shouldn't we just reset the token if it fails at all?